app/views/comments/_commentable.html.erb

```diff
@@ -105,6 +105,10 @@
     <p class="notice">
       <%= t(".blocked") %>
     </p>
+  <% elsif logged_in_as_admin? %>
+    <p class="notice">
+      <%= t("admins.index.log_out_reminder") %>
```

Comment:
How about a new message that says: "Please log out of your admin account to comment."